applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -56,11 +56,20 @@ CONFIG_SLM_AT_MAX_PARAM - AT command parameter count limit
 
 .. _CONFIG_SLM_NATIVE_TLS:
 
-CONFIG_SLM_NATIVE_TLS - Use Zephyr mbedTLS
-   This option enables using Zephyr's mbedTLS.
-   It requires additional configuration.
+CONFIG_SLM_NATIVE_TLS - Use Zephyr's Mbed TLS for TLS connections
+   This option is enabled by the Native TLS overlay.
    See :ref:`slm_native_tls` for more information.
 
+CONFIG_SLM_NATIVE_TLS_CREDENTIAL_BUFFER_SIZE - Buffer space reserved for loading credentials
+   Specifies the credential buffer size available for a single sec_tag when loading credentials for Mbed TLS.
```

Comment:
```suggestion
   Specifies the credential buffer size available for a single ``sec_tag`` when loading credentials for Mbed TLS.
```
Or should it be `<sec_tag>` like it is given in the AT command documentation?

Reply:
Likely it should be ``sec_tag`` when not used as a parameter.